routes/websocket.js

```diff
@@ -252,7 +287,10 @@ export default function init(server, wss) {
 					competition: request.nc_url.searchParams.get('competition') === '1',
 				});
 			}
-		} else if (pathname.startsWith('/ws/room/admin')) {
+		} else if (
+			request.is_secret_admin ||
+			pathname.startsWith('/ws/room/admin')
+		) {
```

Comment:
This change is unnecessary, the `secret_admin` has the same route prefix, so the previous test was good enough